Raise armis event collector default limit (#35371)
demisto/content: Packs/Armis/Integrations/ArmisEventCollector/ArmisEventCollector_description.md

1. Log into the Armis platform and browse to **Settings** by clicking your account icon on the top right-hand side of the screen.
2. Choose **Settings API Management**.
3. Click **Create** and copy the generated key. (Do not share this key and do not create a non-encrypted copy of it.)
4. Refer to [Obtaining an API key from Armis](https://docs.ic.armis.com/docs/introduction_api-keys) for more details.

## General note:

> - Activities & Alerts events types are expected to have a lot of logs in a short interval, therefore the default limit is 100k and the interval is 1 minute.
> - Devices event type is expected to have a heavier response but with least events in a longer interval, therefore the default limit is 50k and the interval is 4 hours.
> If you encounter timeout and internal server errors while fetching events, we suggest first trying to separate instances for each event type and change the limits according to the issues - lowering the limit for timeout or raising the limit for internal server errors.

Reviewer: ```suggestion
- The **Activities** and **Alerts** event types are expected to have a many logs within a short interval. Therefore, the default limit is 100k and the interval is 1 minute.
- The **Devices** event type is expected to have a heavier response but with least events in a longer interval, therefore the default limit is 50k and the interval is 4 hours.
If you encounter timeout and internal server errors while fetching events, separate instances for each event type and tweak the limits according to the issues - lowering the limit for timeout or raising the limit for internal server errors.
```
add a note how too low a limit might cause the time gap to grow, ultimately resulting in internal server errors by the api side

1. Log into the Armis platform and browse to **Settings** by clicking your account icon on the top right-hand side of the screen.
2. Choose **Settings API Management**.
3. Click **Create** and copy the generated key. (Do not share this key and do not create a non-encrypted copy of it.)
4. Refer to [Obtaining an API key from Armis](https://docs.ic.armis.com/docs/introduction_api-keys) for more details.

## General note:

> - The **Activities** and **Alerts** event types are expected to have a many logs within a short interval. Therefore, the default limit is 100k and the interval is 1 minute.
> - The **Devices** event type is expected to have a heavier response but with least events in a longer interval, therefore the default limit is 50k and the interval is 4 hours.
> - internal server errors can occur when there's a growing gap between the available events to fetch and the actual number of events being fetched every minute.
> If you encounter timeout or internal server errors while fetching events, separate instances for each event type and tweak the limits according to the issues - lowering the limit for timeout or raising the limit for internal server errors.

Reviewer: ```suggestion
- The **Activities** and **Alerts** event types are expected to have many logs within a short interval. Therefore, the default limit is 100k and the interval is 1 minute.
- The **Devices** event type is expected to have heavier responses but with fewer events within a long interval. Therefore the default limit is 50k and the interval is 4 hours.
- Internal server errors can occur when there's a growing gap between the available events to fetch and the actual number of events being fetched every minute.
If you encounter timeout or internal server errors while fetching events, separate instances for each event type and tweak the limits according to the issues - lowering the limit for timeout or raising the limit for internal server errors.
```

Reviewer: - Internal server errors can occur when there's a growing gap between the available events to fetch and the actual number of events being fetched every minute.

this sounds not so clear to me but i dont have a clearer way to phrase it

Reviewer: `nternal server errors may occur when there is a significant disparity between the number of events being fetched and the available events within a given time frame. This can happen when the limit set for fetching events is too low, resulting in the retrieval of older events while a substantial number of new events are available.`
chatgpt
what do you think

Reviewer: i would replace that point with this, sounds good to me
